core/deployment/src/main/java/io/quarkus/deployment/JniProcessor.java

```diff
@@ -0,0 +1,37 @@
+package io.quarkus.deployment;
+
+import java.util.List;
+
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.JniBuildItem;
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+public class JniProcessor {
+
+    JniConfig jni;
+
+    @ConfigRoot(phase = ConfigPhase.BUILD_TIME)
+    static class JniConfig {
+        /**
+         * Path of library to load.
+         */
+        @ConfigItem(name = "libraryPaths")
+        List<String> libraryPaths;
+
+        /**
+         * Enable JNI support.
+         */
+        @ConfigItem(name = "enable", defaultValue = "false")
```

Comment:
I think there is no need for `name` since it is the same as field name. It will be auto picked from the field name.

Reply:
You are right ;o)
by default it's `ConfigItem.HYPHENATED_ELEMENT_NAME` which is a the field name -> camel case + hyphnated. Thanks now I discover the process `ConfigDefinition.processConfigGroup`